### PR TITLE
Rename README to README.md

### PR DESCRIPTION
**Pull request by jakobod**
# Contribution description

I changed the name of the `README` to `README.md`. Github will now interpret the `README.md` in a proper way and display a nicer repo-overview. 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +106,7 @@ RTR-Server using the rtrclient command line tool:
 
     bin/rtrclient tcp rpki-validator.realmv6.org 8282
 
-rpki-validator.realmv6.org is an open RTR-Server instance for testing
+`rpki-validator.realmv6.org` is an open RTR-Server instance for testing
 purposes, which runs the RIPE Validator. It listens on port 8282 and
```
Comment (smlng) [inline README.md]: maybe change to 8283 here to use validator 3 instead of 2?

File: README.md [hunk 2]
```diff
@@ -126,8 +128,3 @@ Website: http://rtrlib.realmv6.org/
 
 Mailing List: rtrlib@googlegroups.com
```
Comment (smlng) [inline README.md]: question: can this be changed to a `mailto` link?!

File: README.md (hunk 1 above)
Comment (waehlisch) [reply README.md]: no, not now.

File: README.md (hunk 2 above)
Comment (jakobod) [reply README.md]: Probably! I will take a quick look into that 👍

**Comment by smlng** [comment]
looks good now, please squash

**Comment by jakobod** [comment]
Done.

**Pull request merged**